traveler-ios-ui/TravelerKitUI/ViewControllers/ParkingQueryViewController.swift

```diff
@@ -86,13 +84,15 @@ public class ParkingQueryViewController: UIViewController {
 
         activityIndicator.startAnimating()
 
+        context.result = nil
         Traveler.searchParkingItems(query, identifier: nil, delegate: self)
     }
 }
 
 extension ParkingQueryViewController: ParkingItemSearchDelegate {
     public func parkingItemSearchDidFailWith(_ error: Error, identifier: AnyHashable?) {
         activityIndicator.stopAnimating()
+        context.result = nil
```

Comment:
There's commented code in line 127 (I can't add a comment there because this wasn't on you). Can you get rid of it if its not important?. Thanks.